test/browser_tests/shared_objects/filter.js

```diff
@@ -16,8 +16,11 @@ var filter = {
 
   searchFilterHideBtn: _getFilterElement( '.m-expandable_cue-close' ),
 
-  searchFilterSubmitBtn: _getFilterElement( 'btn[type="submit"]' )
+  searchCategoryLabel: _getFilterElement(
```

Comment:
Possibly? I think we need to rework this list of checkboxes anyway, they should be in a `ul`, but that's out of the scope of this PR.